Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -429,18 +443,31 @@ void Wiimote::SendDataReport()
     // Acceleration:
     if (rpt_builder.HasAccel())
     {
+      Common::Vec3 acceleration;
+      if (m_motionsource == MotionSource::IMU)
+        acceleration = m_imu_accelerometer->GetState();
```

Comment:
I haven't looked at you're UI (I'm not sure if you hide the emulated mappings when configuring a "real" source) but it would useful to somewhat combine "emulated" and "real" accelerometer data. Even just adding on `m_shake_state.acceleration` would allow users to still have a "shake" button. That might be the only practical combination but it's definitely a useful and an easy one to implement.

Reply:
Yes, the emulated mappings are hidden when configuring a "real" source. I considered the scenario of being able to combine motion sources (like in your example), but found that the UI would get messy. Maybe a good UI can be imagined with some contemplation, but I'd prefer to keep it as is for now.

Reply:
Is it OK to leave it as is for now?

Reply:
Yes.